effects/butterfly-gears/data/gen-unrolled-loop.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
```

Comment:
Please remember that [style-python](https://github.com/cahirwpz/demoscene/blob/master/style-python) script will run over our sources and complain if python code is misformated.

Reply:
fixed